components/atoms/VSpinner/VSpinner.vue

```diff
@@ -1,26 +1,31 @@
 <script setup lang="ts">
-withDefaults(
-    defineProps<{
-        playState?: 'paused' | 'running'
-    }>(),
-    {
-        playState: 'running',
-    },
-)
+const props = defineProps<{
+    playState?: 'paused' | 'running'
+    size?: number
+}>()
+
+const boxSize = computed(() => props.size || 50)
```

Comment:
`boxSize` => `size` more concise?

Reply:
Better to use default prop value, I chose withDefaults rather than propsDestructuring.
PropsDestructuring don't keep '.value' syntax, It will be more complicated to debug reactivity chain.